compiler/luci-interpreter/src/loader/KernelBuilder.h

```diff
@@ -83,6 +83,7 @@ class KernelBuilder : public luci::CircleNodeVisitor<std::unique_ptr<Kernel>>
   std::unique_ptr<Kernel> visit(const luci::CirclePack *node) override;
   std::unique_ptr<Kernel> visit(const luci::CirclePad *node) override;
   std::unique_ptr<Kernel> visit(const luci::CirclePadV2 *node) override;
+  std::unique_ptr<Kernel> visit(const luci::CircleMirrorPad *node) override;
```

Comment:
@binarman , this doesn't seem to be in alphabetical order. Are you going to ignore the order afterwards or is this just a mistake?

Reply:
> is this just a mistake?

Yes, I've forgot this list is sorted =(